From d0fcc73c2ea3cfcb57c35b95d6461ad0ce47a082 Mon Sep 17 00:00:00 2001
From: "Reddy, Alavala Srinivasa" <alavala.srinivasa.reddy@intel.com>
Date: Tue, 17 Oct 2023 09:29:08 +0530
Subject: [PATCH] Add isVisible public API.

Task visibilty info will help us decide if correspoding
LG window should be closed or not.

This patch also adds TASK permission to ClipboardAgent
private app.

Change-Id: Ibf4c4153ae3bb114fd65e247cb98d26da772f3ac
Signed-off-by: ahs <amrita.h.s@intel.com>
---
 data/etc/privapp-permissions-platform.xml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/data/etc/privapp-permissions-platform.xml b/data/etc/privapp-permissions-platform.xml
index b6771b7aa855..3e039c703701 100644
--- a/data/etc/privapp-permissions-platform.xml
+++ b/data/etc/privapp-permissions-platform.xml
@@ -114,6 +114,7 @@ applications that come with the platform
         <permission name="android.permission.UPDATE_APP_OPS_STATS"/>
         <permission name="android.permission.SUBSTITUTE_NOTIFICATION_APP_NAME"/>
         <permission name="android.permission.PACKAGE_USAGE_STATS"/>
+        <permission name="android.permission.GET_TASKS"/>
     </privapp-permissions>
 
     <privapp-permissions package="com.android.phone">
-- 
2.17.1

